nixnet/_enums.py

```diff
@@ -1093,6 +1093,71 @@ class ObjectClass(enum.Enum):
 
 
 class CreateSessionMode(enum.Enum):
+    """
+    Frame/Signal list syntax:
+        SIGNAL_IN_SINGLE_POINT,
+        SIGNAL_OUT_SINGLE_POINT: List contains one or more XNET Signal names.
+            Each name must be one of the following options, whichever uniquely
+            identifies a signal within the database given:
+                -<Signal>
+                -<Frame>.<Signal>
+                -<Cluster>.<Frame>.<Signal>
+                -<PDU>.<Signal>
+                -<Cluster>.<PDU>.<Signal>
+            List may also contain one or more trigger signals. For information
+            about trigger signals, refer to Signal Output Single-Point Mode or
+            Signal Input Single-Point Mode.
+
```

Comment:
While I think its useful to move some of the documentation from the parameter descriptions into the enum docstrings, I feel this is a bit too much with the fact that it is covering how individual parameters behave rather than general impact.